src/main/kotlin/com/wire/bots/polls/dao/DaoConstants.kt

```diff
@@ -0,0 +1,5 @@
+package com.wire.bots.polls.dao
+
+const val UUID_LENGTH = 36
+
+const val OPTION_LENGTH = 256
```

Comment:
UUID_LENGTH is common to many classes so it's here, all good (maybe it can have `internal` access modifier?)
If OPTION_LENGTH is related just to a single class, then you can move it to that class/object directly as a const or inside a companion object (and make it private)